src/transformers/models/auto/modeling_auto.py

```diff
@@ -1011,6 +1011,36 @@
     ]
 )
 
+MODEL_FOR_TEXT_ENCODING_MAPPING_NAMES = OrderedDict(
```

Comment:
Is this list good for you @amyeroberts? It's the most popular encoder-only architectures, and enc-dec ones where we have the encoder model separated as an import

Reply:
Yep! We can always add as needed :)